SPI.h should be included in MCP492X.h

Great work, simple API, I like it. 

Almost worked the first try, beside the fact that SPI.h is not included from MCP492X.h. I think that it should be. As a "end user", it is natural to me that the library I use include the libraries it needs without having to think about it.

Other small thing in the example (in the readme.md) there is a small error:
        MCP492X myDac(PIN_SPI_CHIP_SELECT); 
should be 
        MCP492X myDac(PIN_SPI_CHIP_SELECT **_DAC**);

I didn't sent much time with those two issues, it was quite obvious.

Thanks

P.S. I am using it with STM32duino [https://github.com/rogerclarkmelbourne/Arduino_STM32](url)
        chip: stm32F103C8  on a Blue Pill board [http://wiki.stm32duino.com/index.php?title=Blue_Pill](url)(cheap chinese board)

Thank you for this @michd . It looks to be exactly what I also needed without me needing to figure out the coding. 

@lumostor thank you for your comments, I am relatively new to electronics, how do I include the SPI.h library? I have tried downloading the library from elsewhere however keep on receiving this error. I suspect the error is due to the missing SPI.h 

Thank you :)

_Arduino: 1.8.5 (Windows 10), Board: "Arduino/Genuino Uno"

In file included from C:\Users\Sashen Naicker\Documents\Arduino\DAC\SimpleDAC\SimpleDAC.ino:1:0:

MCP492X.h:83: error: 'SPISettings' does not name a type

     SPISettings _spiSettings;

     ^

exit status 1
'SPISettings' does not name a type

This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences._

--- a/MCP492X.h
+++ b/MCP492X.h
@@ -23,6 +23,7 @@
  */
 
 #include <Arduino.h>
+#include <SPI.h>
 
 // Ensure we don't double-define the functionality
 #ifndef MCP492X_h

